### ISSUE TITLE
Need to update "highcharts-react-official": "^3.0.0"

### ISSUE THREAD

**Issue by v-abkhot** [highcharts/highcharts-react#212]
unable to resolved accessibility issue with this version like focus got lost after export menu

**Comment by v-abkhot** [comment]
also focus is getting trapped in graph portion.

**Comment by ppotaczek** [comment]
Hi @v-abkhot,

I did not notice any problems with accessibility in our wrapper. Please provide me with an example and more detailed description of the issue.

You can start from: https://codesandbox.io/s/suspicious-mcclintock-7bm4u

Best regards!

**Comment by v-abkhot** [comment]
![image](https://user-images.githubusercontent.com/63849538/79569273-f3354c80-80d4-11ea-94b3-14aa560be755.png)


if you see in this image focus after 1st graph it is directly go to that right corner dropdown

**Comment by v-abkhot** [comment]
so can you please provide some inputs on this ?

**Comment by ppotaczek** [comment]
Hi @v-abkhot,

That issue is not related with this wrapper, but probably with Highcharts. Please report it on the main repository: https://github.com/highcharts/highcharts/issues

Best regards!
